Project-Euler/Project-Euler/Program.cs

```diff
@@ -20,19 +37,61 @@ public static void PrintIntro()
             //Add a couple more lines, maybe?
         }
 
-        public static void PrintMenuOptions() //This method will be private once I write the method that will call it. blah blah
+        public static int GetMenuSelection()
+        {
+            PrintMenuOptions();
```

Comment:
Instead of putting _PrintMenuOptions_ in 2 places (here and at the bottom of the while loop), just put it at the top of the while loop